lib/phases/check.js

```diff
@@ -23,7 +23,13 @@ exports.checkServices = function (serviceDeployers, environmentContext) {
     let errors = [];
     _.forEach(environmentContext.serviceContexts, function (serviceContext) {
         let serviceDeployer = serviceDeployers[serviceContext.serviceType];
-        let checkErrors = serviceDeployer.check(serviceContext);
+        let dependenciesServiceContexts = [];
```

Comment:
Maybe split this out into a little function in this file called "getDependenciesServiceContexts". That will help reduce the noise in this function.